### ISSUE TITLE
Extensions to other currencies + some questions

### ISSUE THREAD

**Issue by mrwillis** [GlacierProtocol/GlacierProtocol#4]
Hi,

This is an awesome document. Thank you. Right now I am looking to extend it for Ethereum using `geth` and essentially copying the protocol and just replacing the `bitcoin-core` stuff with the relevant `geth` commands (and I guess smart contracts). Would you accept a PR for this? Do you think there would be any interesting complications that might arise?

I also have a question about the protocol:

Is there any benefit to having one of the host computers dedicated to this process? I.e., all it does is connect to WiFi and download packages relevant to the protocol? Or is this pointless because either
- the use of _two_ host computers already basically eliminates the possibility that the same malware is installed on both, hence the use of a dedicated PC as one off the PCs is just as good as any PC as long as we can be reasonably sure that not both PCs are badly infected
- even on a malware infected PC, there is little chance that the malware could infect the image to be installed on the boot/app USB or the USB provided the checksum is verified. I guess this is really just an extra safeguard and the does not replace the use of two host PCs. Is this right?

Thanks for helping me understand.

**Comment by diogomonica** [comment]
@mrwillis we would definitely accept a PR to add support for ETH, thank you for offering. We're looking to make the following changes to the protocol:

- Bring the PDF into offline HTML + Javascript.
- Add QR-code generation to the page so that it reduces the installation steps needed from glacier and reduces the time to execute the protocol.
- Add ETH support at the same level of security as BTC.
- Bring in more contributions from the community.

As far as your question goes, it's more of the latter: if you are using eternally quarantined hardware like the protocol recommends, and ensuring the integrity of the OS that you're loading the hardware from (reputable source, hasn't been modified at rest), there isn't much surface of exposure to malware doing  any harm.

**Comment by bitcoinhodler** [comment]
Request: if adding ETH (or other altcoin) support, ensure it doesn't complicate the BTC experience. For example, I shouldn't have to repeatedly skip over ETH-specific instructions if I'm doing BTC, and vice versa. Too confusing and too many opportunities for mistakes in execution. Instead, ask the user which coin(s) they want, then present instructions for only that coin.

**Comment by another-hodler** [comment]
I agree with @bitcoinhodler. For different currency there should be completely separate protocol. This protocol is meant for long term storage and therefore protocol stability should be the highest priority. The larger the document is the bigger the number of issues can be raised and the more frequent release cycle is required.
